compiler/compiler.go

```diff
@@ -13,16 +13,16 @@ import (
 
 // ParseProc() is an entry point for use from external go code,
 // mostly just a wrapper around Parse() that casts the return value.
-func ParseProc(query string, opts ...zql.Option) (ast.Proc, error) {
-	parsed, err := zql.Parse("", []byte(query), opts...)
+func ParseProc(z string) (ast.Proc, error) {
+	parsed, err := zql.ParseZ(z)
 	if err != nil {
-		return nil, err
+		return nil, zql.ImproveError(z, err)
```

Comment:
Don't want this with zql.ParseZ.
```suggestion
		return nil, err
```

Reply:
I'm getting old, haha